landoui/templates/stack/stack.html

```diff
@@ -187,5 +175,60 @@ <h2>Landing is blocked:</h2>
     </div>
   </div>
 
+  {% if is_user_authenticated() %}
```

Comment:
Should this not match the same conditions as the other if statement on line 123? Also, could set those two in a variable to avoid repetition (e.g., `meets_uplift_conditions`)